### PR TITLE
fix: add copies and fix card height

### PR DESCRIPTION
**Pull request by ioanaroceanu2000**

### CODE REVIEW

File: src/components/interface/MellowLpDepositForm/components/LPMellowVaultDepositWindow.tsx [hunk 1]
```diff
@@ -40,7 +40,13 @@ const LPMellowVaultDepositWindow: React.FunctionComponent<LPMellowVaultDepositWi
 
   return (
     <FormPanel
-      sx={{ width: '100%', maxWidth: '328px', background: '#19152A', padding: '16px 16px' }}
+      sx={{
+        width: '100%',
+        maxWidth: '328px',
+        maxHeight: '444px',
+        background: '#19152A',
+        padding: '16px',
```
Comment (ssbarbee) [inline src/components/interface/MellowLpDepositForm/components/LPMellowVaultDepositWindow.tsx:48]:
```suggestion
        padding: (theme) => theme.spacing(4)
```

File: src/components/interface/MellowLpDepositForm/components/LPMellowVaultDepositWindow.tsx [hunk 2]
```diff
@@ -78,15 +84,15 @@ const LPMellowVaultDepositWindow: React.FunctionComponent<LPMellowVaultDepositWi
...
         variant="body1"
         sx={{
-          fontSize: '16px',
+          fontFamily: 'DM Sans',
+          fontSize: '14px',
           lineHeight: '1.6',
-          color: '#E5E1F9',
-          marginTop: '8px',
+          color: '#e5e1f9d9',
+          marginTop: '16px',
```
Comment (ssbarbee) [inline src/components/interface/MellowLpDepositForm/components/LPMellowVaultDepositWindow.tsx:91]:
```suggestion
          marginTop: (theme) => theme.spacing(4),
```

File: src/components/interface/MellowLpDepositForm/components/LPMellowVaultDepositWindow.tsx [hunk 3]
```diff
@@ -146,21 +152,26 @@ const LPMellowVaultDepositWindow: React.FunctionComponent<LPMellowVaultDepositWi
...
         >
           ABOUT YOUR FUNDS
         </Typography>
         <Typography
           variant="body1"
-          sx={{ fontSize: '12px', color: '#9B97AD', marginTop: '10px', fontWeight: '400' }}
+          sx={{
+            fontSize: '12px',
+            color: '#9B97AD',
+            marginTop: '10px',
```
Comment (ssbarbee) [inline src/components/interface/MellowLpDepositForm/components/LPMellowVaultDepositWindow.tsx:168]:
```suggestion
            marginTop: (theme) => theme.spacing(2.5),
```

File: src/components/interface/MellowLpDepositForm/components/LPMellowVaultDepositWindow.tsx (hunk 1 above)
Comment (ioanaroceanu2000) [reply src/components/interface/MellowLpDepositForm/components/LPMellowVaultDepositWindow.tsx]: fixed in a880d81b18fc358da6db21e427119ac7f03e85ef

File: src/components/interface/MellowLpDepositForm/components/LPMellowVaultDepositWindow.tsx (hunk 2 above)
Comment (ioanaroceanu2000) [reply src/components/interface/MellowLpDepositForm/components/LPMellowVaultDepositWindow.tsx]: fixed in a880d81b18fc358da6db21e427119ac7f03e85ef

File: src/components/interface/MellowLpDepositForm/components/LPMellowVaultDepositWindow.tsx (hunk 3 above)
Comment (ioanaroceanu2000) [reply src/components/interface/MellowLpDepositForm/components/LPMellowVaultDepositWindow.tsx]: fixed in a880d81b18fc358da6db21e427119ac7f03e85ef

**Review by ssbarbee — APPROVED** [review]

**Pull request merged**